<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="Setting_StackedWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Format_Page">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Format: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Format_LineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="whatsThis">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use &amp;quot;%&amp;quot; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>?</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Axis_Page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>X Axis:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="X_AxisLowRange_LineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use &amp;quot;%&amp;quot; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="X_AxisHighRange_LineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Y Axis:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Y_AxisLowRange_LineEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use &amp;quot;%&amp;quot; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Y_AxisHighRange_LineEdit_2"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="Graph_Widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Format"/>
   <addaction name="action_Axis"/>
   <addaction name="separator"/>
   <addaction name="action_SaveGraph"/>
  </widget>
  <action name="action_Format">
   <property name="text">
    <string>Format</string>
   </property>
  </action>
  <action name="action_Axis">
   <property name="text">
    <string>Axis</string>
   </property>
  </action>
  <action name="action_SaveGraph">
   <property name="icon">
    <iconset resource="../Icons/Icons.qrc">
     <normaloff>:/Save_2.png</normaloff>:/Save_2.png</iconset>
   </property>
   <property name="text">
    <string>SaveGraph</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Icons/Icons.qrc"/>
  <include location="../Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
